Sasquatch/SasquatchObjC/AppDelegate.m

```diff
@@ -128,6 +130,12 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     break;
   }
 
+  // Setup location manager.
+  self.locationManager = [[CLLocationManager alloc] init];
+  self.locationManager.delegate = self;
+  self.locationManager.desiredAccuracy = kCLLocationAccuracyKilometer;
+  [self.locationManager requestWhenInUseAuthorization];
```

Comment:
Suggestion: request only when needed instead of on application start

Reply:
Good point but this is a test app so we can change this later. Not merge blocker.